src/testdrive/src/action/kafka/verify.rs

```diff
@@ -273,3 +274,98 @@ impl Action for VerifyAction {
         }
     }
 }
+
+pub struct VerifySchemaAction {
+    sink: String,
+    format: SinkFormat,
+    expected_key_schema: Option<String>,
+    expected_value_schema: String,
+    context: Context,
+}
+
+pub fn build_verify_schema(mut cmd: BuiltinCommand, context: Context) -> Result<VerifySchemaAction, String> {
+    let format = match cmd.args.string("format")?.as_str() {
+        "avro" => SinkFormat::Avro,
+        "json" => SinkFormat::Json {
+            key: cmd.args.parse("key")?,
+        },
+        f => return Err(format!("unknown format: {}", f)),
+    };
+    let sink = cmd.args.string("sink")?;
+
+    let (key, value) : (Option<String>, String ) = match &cmd.input[..] {
+        [value] => (None, value.clone()),
+        [key, value] => (Some(key.clone()), value.clone()),
+        _ => return Err(String::from("unable to read key/value schema inputs"))
+    };
+
```

Comment:
There are a few extra spaces here. You should run `cargo fmt`, otherwise your change will be rejected by CI.